docker-entrypoint.sh

```diff
@@ -40,9 +40,13 @@ if (is_numeric($socket)) {
 	$socket = null;
 }
 
-$maxTries = 10;
+$maxTries = 20;
```

Comment:
What is the reason behind this change?

Reply:
This doesn't have to change.  It just gives the container more time alive before crashing and stopping which makes it a little easier to troubleshoot.

Assuming people aren't running into issues it can be set to 10, or even 4

Reply:
Then I would prefer not to change the value 🙂 

```suggestion
$maxTries = 10;
```